### DISCUSSION TITLE
UI keeps getting messed up whenever an event occurs

### DISCUSSION THREAD

**Discussion by acgonzales** [livewire/livewire#3379] [1. Help]
This is my first project using Livewire. I like it for the most part but I'm having issues especially on [DOM diffing](https://laravel-livewire.com/docs/2.x/troubleshooting) . I have elements that have their UI messed up and can't be interacted with whenever an event occurs. Most of the time I was able to fix it by placing a `wire:ignore` directive on the element. But this is the part that I can't fix.

I have the component `profile-body` which renders the current active tab.

```html
<div class="row profile-body">
    <div class="col-sm-12">
        @switch($currentTab)
        @case(0)
        <livewire:profile.forms.basic-information-form />
        @break
        @case(1)
        <livewire:profile.forms.addresses-form />
        @break
        @endswitch
    </div>
</div>
```

And the elements and inputs in the `forms.basic-information-form` becomes non-interactive when I switch to the `forms.addresses-form` and back.

It fits the DOM diffing issues symptons stated in the [documentation](https://laravel-livewire.com/docs/2.x/troubleshooting). I tried to perform the recommended "cures" for it like adding a key, or wrapping it in an element. But still, no luck. Any helps guys?

**Comment by jamesbhatta** [comment]
Have you tried wrapping up ``<livewire:profile .... />`` in a ``div`` and adding ``wire:key`` to it?

**Reply by acgonzales** [reply]
Yes, I already tried that but still the same. I decided to not use livewire for this project anymore.
